src/core/split_multimesh_container.gd

```diff
@@ -0,0 +1,47 @@
+class_name SplitMultimeshContainer
+extends Spatial
+
+var visible_range_begin : float = 0
+var visible_range_begin_hysteresis : float = 0.1
+var visible_range_end : float   = 0
+var visible_range_end_hysteresis : float = 0.1
+
+func _ready():
+	pass
```

Comment:
In _ready, you should check if the feature is enabled or not, if both begin and end values are 0, call `set_process(false)` otherwise it will still needlessly loop through all children every frame.

Reply:
I'm not sure why but this approach disables the process even if the visible range is not zero. How about returning at the start of the process function? That way the script doesn't need to loop over the children and it works well.

Reply:
Maybe that's because in _ready the visible_range is still set to zero, and then it gets changed by the scatter node on the first update. So by the time the actual values are set, `set_process(false)` was already called.

Reply:
But yeah, returning early at the beginning of _process works too.